CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [2.2.1] - 2020-02-04
+
+### Added
+- Metrics to the `WordPress.Arrays.CommaAfterArrayItem` sniff. These can be displayed using `--report=info`.
+- The `sanitize_hex_color()` and the `sanitize_hex_color()` functions to the `escapingFunctions` list used by the `WordPress.Security.EscapeOutput` sniff.
```

Comment:
```suggestion
- The `sanitize_hex_color()` and the `sanitize_hex_color_no_hash()` functions to the `escapingFunctions` list used by the `WordPress.Security.EscapeOutput` sniff.
```

Reply:
:fearful: copy/paste fail... Thanks for catching that. Updated now.